plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBaselineSupportTest.kt

```diff
@@ -135,6 +135,7 @@ class KtlintBaselineSupportTest : AbstractPluginTest() {
                 buildGradle.appendText(
                     """
                     ktlint.version = "$ktLintVersion"
+                    ktlint.debug = true
```

Comment:
Very sane, why we haven't done this before, I don't know 😂

Reply:
hah I needed this to debug some issues. I could remove it, but I guess we might as well leave this here otherwise we will probably forget about that option.